### PR TITLE
EWL-1048 - Add ellipsis to StyledPlayerName

### PR DESCRIPTION
**Pull request by degouville**
# About the type of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Test plan

- [ ] Empty data
- [ ] Invalid data
- [x] Correct data

# Checklist
### Before code review

- [x] Move Jira ticket into PR
- [x] Check with another developers that features sounds good

### After code review

- [x] Move Jira ticket into QA

### After QA validation
If conflicts are present in github:
  - [ ] `git pull --rebase origin master`
  - [ ] Fix conflicts
  - [ ] `git rebase --continue` repeat until there is no more conflicts
  - [ ] `git push --force-with-lease`

If there's no conflicts
- [ ] squash and merge
- [ ] delete branch

### CODE REVIEW

File: src/components/ScoreBlock/SetsScore.js [hunk 1]
```diff
@@ -27,6 +27,18 @@ export const StyledPlayer = styled.div`
 
 const StyledPlayerName = styled.div`
   display: block;
+
+  text-overflow: ellipsis;
```
Comment (davidgarry) [inline src/components/ScoreBlock/SetsScore.js]: ⚠️ Your ellipsis hides the winner / service icon. It should not hide them.

Comment (degouville) [reply src/components/ScoreBlock/SetsScore.js]: FIXED ✅

Comment (davidgarry) [reply src/components/ScoreBlock/SetsScore.js]: I think something is broken on large devices :
image

Comment (degouville) [reply src/components/ScoreBlock/SetsScore.js]: FIXED ✅

**Comment by davidgarry** [comment]
When the width is too small, the scores are not aligned anymore (because of the service icon)
image

**Pull request closed without merging**